src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/ITextDocumentTextPresentationHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using OmniSharp.Extensions.JsonRpc;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.DocumentPresentation
+{
+    [Parallel, Method(VSInternalMethods.TextDocumentTextPresentationName)]
+    internal interface ITextDocumentTextPresentationHandler : IJsonRpcRequestHandler<TextPresentationParams, WorkspaceEdit?>, IRegistrationExtension
```

Comment:
Problem number 3: In a hilarious addendum to problem number 1, I never actually got around to even creating a TextPresentation handler